gspread/client.py

```diff
@@ -38,20 +31,16 @@ class Client(object):
 
     """
     def __init__(self, auth, session=None):
-        self.auth = auth
-        self.session = session or requests.Session()
+        self.auth = convert_credentials(auth)
+        self.session = session or AuthorizedSession(self.auth)
```

Comment:
AuthorizedSession is not defined

Reply:
I'm probably overlooking something, but all I've tried so far is using https://developers.google.com/sheets/api/quickstart/python , returning `creds`, and using this PR instead of normal gspread, like so:

`gspread.authorize(creds)`

Reply:
adding `from google.auth.transport.requests import AuthorizedSession` at the top of `def __init__` seems to fix it.

Reply:
Hey, thanks for reviewing! I should have probably re-checked this before submitting. You're right, I was just missing the import. I've added it now and it should be working.